src/aiomealie/models.py

```diff
@@ -81,6 +82,19 @@ class RecipesResponse(DataClassORJSONMixin):
     items: list[Recipe]
 
 
+class OptionalStringSerializationStrategy(SerializationStrategy):
+    """Serialization strategy for optional strings."""
+
+    def serialize(self, value: str | None) -> str | None:
+        """Serialize optional string."""
+        return value
```

Comment:
This is not used internally and would only be used by external usage, looks trivial so no unit tests here